syntax_and_semantics/capturing_blocks.md

```diff
@@ -37,29 +41,91 @@ model.save # prints "Saved!"
 
 In the above example the type of `&block` wasn't specified: this just means that the captured block doesn't have arguments and doesn't return anything.
 
-Note that if the return type is not specified, nothing gets returned from the proc call:
+## Options for capturing blocks
+
+Capturing is not limited to requiring one block argument and a specific return type.
+A block can be captured in many ways. A captured block can:
```

Comment:
I think these two lines should be rephrased, because they are not really precise and easy to understand. These type restrictions do not only apply for capturing blocks, they also apply for non-captured blocks (the only difference is that with non-captured you can omit them).
So I'm not sure if this entire section fits here.

Reply:
Is there a better spot for it?

If it belongs somewhere else, I would want a link to it from this page - so you can figure out the different ways that are permissible to capture blocks from this page (even if through a link)

Reply:
I guess there is no place for "blocks in general". So it probably might as well stay here...